docs/src/ann_indexes.md

```diff
@@ -121,7 +122,9 @@ You can select the columns returned by the query using a select clause.
 
 === "Python"
      ```python
-     tbl.search(np.random.random((768))).select(["vector"]).to_df()
+     tbl.search(np.random.random((1536))).select(["vector"]).to_df()
+     ```
+     ```
```

Comment:
```suggestion
```

Reply:
I think we might want to keep this to maintain the next code block: 
<img width="691" alt="Screen Shot 2023-06-22 at 10 15 12 AM" src="https://github.com/lancedb/lancedb/assets/43354492/4e4542ed-d83f-48ca-87fb-d17332a3c04e">